yogstation/code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -0,0 +1,8 @@
+/mob/living/silicon/ai/proc/handle_remotedoor(href_list)
+	var/obj/machinery/door/airlock/A = locate(href_list["remotedoor"]) in GLOB.machines
```

Comment:
Href exploit here, add `if(src != usr) return` to check if it was the AI who called this